src/BernardPublisher.php

```diff
@@ -15,7 +15,7 @@ class BernardPublisher implements Publisher
     private $queueResolver;
     private $type;
 
-    public function __construct(MessageInEnvelopSerializer $serializer, Producer $bernard, RoutingKeyResolver $queueResolver, $type)
+    public function __construct(MessageInEnvelopeSerializer $serializer, Producer $bernard, RoutingKeyResolver $queueResolver, $type)
```

Comment:
With the drop of the old PHP versions i'm looking into releasing al new marjor versions I think in that case this wound be a problem right?